pkg/kubernetes/tool.html

```diff
@@ -294,6 +361,16 @@ <h5 class="list-group-item-heading">{{id}}</h5>
     {{/desiredState}}
   </script>
 
+  <script id="add-dialog-errorMsg-tmpl" type="x-template/mustache">
```

Comment:
You don't need a template for this. Just Add this to the 'add-dialog-fail' in the dialog below. And then set the hidden attribute on it. You'll need to .hide() it when the dialog is shown, and add an id='' to the <strong> tag, so you can fill in the text.